dafni_cli/get.py

```diff
@@ -0,0 +1,67 @@
+import click
+
+from dafni_cli.login import check_for_jwt_file
+from dafni_cli.API_requests import get_models_dicts
+from dafni_cli.model import create_model_list
+
+
+@click.group()
+@click.pass_context
+def get(ctx):
+    """Lists entities available to the user from
+    models, datasets, workflows, groups, depending on command."""
+    ctx.ensure_object(dict)
+    jwt_dict, jwt_flag = check_for_jwt_file()
```

Comment:
can we do click.invoke(login) here instead so we get the associated echos to the command line

Reply:
That's what I originally had it as, but I changed it because I didn't know whether it would be annoying to be told your username and uuid every time a user does anything. If they aren't logged in, then the prompts and the "Login successful, user, uuid" is still printed, but doesn't show anything if they are already logged in. Whereas if they are already logged in and use the `login` command, the "you are already logged in as <username>, <uuid>" is printed.

Can change it back if you disagree though.

Reply:
That sounds sensible